src/test/java/com/feedzai/commons/sql/abstraction/engine/impl/oracle/OracleEngineSchemaTest.java

```diff
@@ -97,10 +112,101 @@ public void testSystemGeneratedColumns() throws Exception {
                     .build();
             engine.addEntity(entity);
 
-            assertFalse("The simulated system generated column should not appear in the table metadata", engine.getMetadata("TEST_SYS_COL").containsKey("SYS_COL1"));
-            assertTrue("The regular column should appear in the table metadata", engine.getMetadata("TEST_SYS_COL").containsKey("COL1"));
-        } finally {
-            engine.close();
+            assertFalse(
+                    "The simulated system generated column should not appear in the table metadata",
+                    engine.getMetadata("TEST_SYS_COL").containsKey("SYS_COL1")
+            );
+            assertTrue(
+                    "The regular column should appear in the table metadata",
+                    engine.getMetadata("TEST_SYS_COL").containsKey("COL1")
+            );
         }
     }
+
+    /**
+     * This method tests that, when the {@link PdbProperties pdb.compress_lobs} is true (which is the default value),
+     * the LOBS columns are compressed: Both BLOB and CLOB types are tested.
+     *
+     * @throws Exception if anything goes wrong with the test
+     */
+    @Test
+    public void testCompressLobs() throws Exception {
+        final String tablespace = "TEST_TABLESPACE";
+        final String tableName = "TEST_TABLE";
+        final String tableUserLobs = "USER_LOBS";
+
+        final String tableId = "ID";
+        final String tableBlob = "BLOB_COLUMN";
+        final String tableClob = "CLOB_COLUMN";
+
+        final String compression = "COMPRESSION";
+        final String secureFile = "SECUREFILE";
+
+        properties.setProperty(COMPRESS_LOBS, Boolean.toString(true));
+
+        final DatabaseEngine engine = DatabaseFactory.getConnection(properties);
+
+        createUserTablespace(tablespace);
```

Comment:
not sure if...
could these _create/updateUserTablespace_ use "engine" intead of creating another one in each method
and maybe both could be joined into just 1 method?

Reply:
Yes, they can. I will do that.